static/js/compose_actions.js

```diff
@@ -132,7 +132,6 @@ export function complete_starting_tasks(msg_type, opts) {
     // makes testing a bit easier.
 
     maybe_scroll_up_selected_message();
-    ui_util.change_tab_to("#message_feed_container");
```

Comment:
This was changing the narrow to all messages view when starting compose, I am not sure what this even does. Removing this fixes the bug. @showell FYI

Reply:
I can relate to this one. The web app goes to all messages when we tried to start a compose box in recent topics using `compose_actions.start`.
I also don't know what it does, but this surely fixes the bug.

Reply:
Removing this is correct.